frontend/src/app/(protected)/experiments/add/components/addExperimentSteps.tsx

```diff
@@ -0,0 +1,30 @@
+import { Step } from "../../types";
+import AddMABArms from "./mabs/addMABArms";
+import AddABArms from "./ab/addABArms";
+import AddNotifications from "./addNotifications";
+
+// --- MAB types and steps ---
+
+const MABsteps: Step[] = [
+  {
+    name: "Add Arms",
+    component: AddMABArms,
+  },
+  { name: "Notifications", component: AddNotifications },
+];
+
+// --- A/B test types and steps ---
```

Comment:
This is where you would define the various "screens" (components) for each experiment type.